apps/e2e/tests/backend/endpoints/api/v1/integrations/custom/oauth.test.ts

```
@@ -151,14 +151,14 @@ async function authorize(projectId: string) {
 }
 
 it(`should redirect to the correct callback URL`, async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)